### DISCUSSION TITLE
How to change font family in PDF

### DISCUSSION THREAD

**Discussion by vuquangtam** [emacs-eaf/emacs-application-framework#923] [Q&A] [answered]
Hi, thanks for awesome product. 

Recently I want to read some books in epub but got an error with font. Text didn't show some characters like ă â ắ ấ... I thought it is the font issue.

Do we have any ways that we can change the font family in pdf. 

Thanks

**Comment by manateelazycat** [comment, marked as answer]
I have add epub to eaf-find-file-ext-blacklist, eaf-pdf-viewer (base on mupdf library) not good for epub file.
